### DISCUSSION TITLE
...

### DISCUSSION THREAD

**Discussion by DaniaNouro** [MuhammadSulaiman001/opengl-lab#26] [Q&A] [answered]
مرحبا استاذ،عندي سؤالين الاول 
انو ماكتير توضحتلي فكرة ليش بدي خلي تابع الmainياخد parameters
بحثت عن الموضوع بس انو ماوصلني شو الهدف الاساسي 
والسؤال التاني

قبل مابدل الفيجوال كانت الامور عندي تمام والمشاريع كلها شغالة "وسألتك عن الايرور وحلينا سوا واشتغلو المشاريع "
اي بعد مابدلتو ماعاد اشتغلو ومو نفس الايرور الماضي 
اذا مااشتغلو ليوم الخميس 
بتروح علاماتهن او في حل ثاني وبس 😁

**Comment by MuhammadSulaiman001** [comment, marked as answer]
اهلا وسهلا،
موضوع انو الmain ياخد parameters هاد أمر شائع جدا بتطبيقات الconsole **وتطبيقات سطح المكتب**، الهدف بشكل عام هو تمرير اعدادات للتطبيق من خارج التطبيق.. بهي الوظيفة بدنا نتحكم بالحالة الابتدائية للرقعة.
بالنسبة للسؤال التاني (والأول كمان)، هي طلبات أصلية بالوظيفة، وكلها ممكنة وغير تعجيزية، وقابلنا فيها 3 مجموعات الخميس الماضي، فما في امكانية انها تتعدل ابدا.
بالتوفيق.
